bind_exporter.go

```diff
@@ -202,6 +202,11 @@ var (
 			"Number of current recursive clients.",
 			nil, nil,
 		),
+		"RPZRewrites": prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "rpz_rewrites"),
```

Comment:
This is a counter and must end in `_total`.

```suggestion
			prometheus.BuildFQName(namespace, "", "response_policy_zone_rewrites_total"),
```